.github/workflows/ci-cd.yml

```diff
@@ -18,22 +18,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: Set up nodejs
         uses: actions/setup-node@v2
```

Comment:
actions/setup-node should be updated to v3

Reply:
Thanks Meli for pointing this out!